renovate.json

```diff
@@ -0,0 +1,21 @@
+{
+  "commitMessageAction": "update",
+  "commitMessagePrefix": "⬆️(dependencies)",
+  "commitMessageTopic": "{{depName}}",
+  "enabledManagers": ["npm"],
+  "extends": ["config:base"],
+  "packageRules": [
+    {
+      "packagePatterns": [ "*" ],
+      "groupName": "allDependencies",
+      "schedule": [ "on the first day of the month" ]
+    }
+  ],
+  "prConcurrentLimit": 2,
+  "prHourlyLimit": 1,
+  "prCreation": "not-pending",
+  "rebaseStalePrs": true,
+  "semanticCommits": false,
+  "separateMajorMinor": false,
```

Comment:
This means you won't get backported minor or patch releases for a dependency from the moment that a new major release is available for it - usually this is not recommended, especially as a global rule.